Lib/test/test_gettext.py

```diff
@@ -937,6 +936,50 @@ def test_lazy_import(self):
         ensure_lazy_imports("gettext", {"re", "warnings", "locale"})
 
 
+class DGettextTest(GettextBaseTest):
+
+    def setUp(self):
+        GettextBaseTest.setUp(self)
+        gettext.bindtextdomain('gettext', os.curdir)
```

Comment:
Don't we need to tearDown this one?

Reply:
Seems `reset_gettext()` resets the state from `GettextBaseTest` is enough